WordPress/src/main/res/values/strings.xml

```diff
@@ -1089,6 +1090,10 @@
     <string name="error_notif_q_action_approve">Could not approve comment. Please try again later.</string>
     <string name="error_notif_q_action_reply">Could not reply to comment. Please try again later.</string>
 
+    <!-- Post Error -->
+    <string name="error_unknown_post">Unknown post, local copy of this post could be desynchronized with you server</string>
```

Comment:
Actually, I think for English speakers `Unknown post, local copy of this post may be out of sync with the server` might make more sense.